sdk/messaging/azservicebus/go.mod

```diff
@@ -3,15 +3,14 @@ module github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus
 go 1.16
 
 require (
-	github.com/Azure/azure-amqp-common-go/v3 v3.2.1
 	github.com/Azure/azure-sdk-for-go v51.1.0+incompatible
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.20.0
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v0.12.0
-	github.com/Azure/azure-sdk-for-go/sdk/internal v0.8.1
+	github.com/Azure/azure-sdk-for-go/sdk/internal v0.8.2
+	github.com/Azure/azure-sdk-for-go/sdk/messaging/internal v0.0.0-20211118043544-abfa4f313a9c
```

Comment:
When will this get tagged?

Reply:
(offline discussion had)

We're going to leave this untagged for now. There are some benefits that seem to apply to this package. Mainly, since this code is not intended to really be used externally (it's just going to be some shared code between eventhubs and servicebus) we don't need to have it show up in pkg.go.dev.

Sharp edge: when we make changes in internal we'll need to push out new SB and EH packages, but this fits well with how those packages will work, so again, probably okay.